internals/cli/cmd_help.go

```diff
@@ -25,10 +25,30 @@ import (
 	"github.com/canonical/go-flags"
 )
 
-var shortHelpHelp = "Show help about a command"
-var longHelpHelp = `
-The help command displays information about commands.
-`
+const cmdHelpSummary = "Show help about a command"
+const cmdHelpDescription = "The help command displays information about commands."
```

Comment:
Same, let's please preserve the breaking consistently as we had before (see diff above).